library/ec2_eip_facts.py

```diff
@@ -75,8 +75,8 @@
 
 '''
 
-from ansible.module_utils.aws.core import AnsibleAWSModule
-from ansible.module_utils.ec2 import (ansible_dict_to_boto3_filter_list,
+from ansible_collections.amazon.aws.plugins.module_utils.core import AnsibleAWSModule
```

Comment:
As also stated [here](https://github.com/Flaconi/ansible-role-aws-vpc-eip/pull/5#pullrequestreview-793235536), I suggest switching to the built-in module.